src/engine/Server.cpp

```diff
@@ -914,18 +965,22 @@ bool Server::checkAccessToken(
 
 // _____________________________________________________________________________
 
-std::optional<std::string_view> Server::checkParameter(
-    const ad_utility::HashMap<std::string, std::string>& parameters,
-    std::string_view key, std::optional<std::string_view> value,
+std::optional<std::string> Server::checkParameter(
+    const ad_utility::url_parser::ParamValueMap& parameters,
+    std::string_view key, std::optional<std::string> value,
```

Comment:
Changed the `string_view` to `string` to make everything work nicely together again.